lib/index.js

```diff
@@ -44,11 +45,28 @@ function qRouter(spec) {
                     const cascade = targets.some(target => {
                         if ('cond' in target) {
                             // Do any of the conditions reference the sectionId
-                            return target.cond.some(element =>
-                                String(element).includes(`.${sectionId}.`)
-                            );
+                            if (target.cond.some(element => String(element).includes('|role'))) {
+                                const roleConditions = target.cond;
+                                for (
+                                    let conditions = 1, len = roleConditions.length;
```

Comment:
Can we find another way to write this for loop?

Reply:
Tidied up loop